src/components/TrendChartDetails.js

```diff
@@ -29,22 +31,22 @@ const TrendChartDetails = ({
   colors,
   crime,
   data,
-  keys,
   since,
   until,
   updateYear,
 }) => {
-  const { name, slug } = keys[0]
-  const comparison = getComparison({ place: slug, data })
-  const rate = data[slug].rate
-  const year = data.date.getFullYear()
+  const keys = uniq(data.map(d => d.place))
+  const isOnlyNational = keys.length === 1
+
+  const comparison = getComparison({ place: keys[0], data })
+  const rate = data[0].rate
```

Comment:
No, I think this is a bad assumption. We should explicitly pull out the item we want based on the state